app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/model/DummyCategorizedPicture.kt

```diff
@@ -0,0 +1,31 @@
+package com.github.HumanLearning2021.HumanLearningApp.model
+
+import android.app.Activity
+import android.widget.ImageView
+import com.github.HumanLearning2021.HumanLearningApp.R
+import java.lang.IllegalArgumentException
+
+/**
+ * A picture part of the dummy data set. Can be of any of the following categories: "fork", "knife", "spoon"
+ */
+class DummyCategorizedPicture(override val category: Category) : CategorizedPicture {
+
+    override fun displayOn(activity: Activity, imageView: ImageView) {
+        if(category !is DummyCategory) throw IllegalArgumentException("provide a dummy category to the class constructor")
+
+        when(category.name){
+            "fork" -> imageView.setImageResource(R.drawable.fork)
+            "knife" -> imageView.setImageResource(R.drawable.knife)
+            "spoon" -> imageView.setImageResource(R.drawable.spoon)
+            else -> throw IllegalArgumentException("only spoon, fork, knife are valid category to retrieve from dummy dataset")
+        }
+    }
+
+    override fun equals(other: Any?): Boolean {
+        return other is CategorizedPicture && other.category == category
+    }
+
+    override fun hashCode(): Int {
+        return 17 + 31*category.hashCode()
+    }
```

Comment:
Is it possible to use a data class in order to avoid having to write these?

Reply:
yes, thanks. done